tests/PolyType.SourceGenerator.UnitTests/Analyzers/TypeShapeExtensionAnalyzerTests.cs

```diff
@@ -24,7 +24,12 @@ public class PointMarshaller : IMarshaller<Point, PointMarshaller.PointSurrogate
 
                 public PointSurrogate ToSurrogate(Point value) => throw new System.NotImplementedException();
 
-                public record struct PointSurrogate(int X, int Y);
+                public struct PointSurrogate
```

Comment:
Is this changed because C# 9 does't support records? We should still be able to up the language version in testing right?